Tools/Frame_params/Solo_AC36.param

```diff
@@ -66,12 +73,21 @@ GPS_NAVFILTER,6
 INS_ACC_BODYFIX,3
 INS_FAST_SAMPLE,3
 INS_TRIM_OPTION,2
+LAND_ALT_LOW,450
 LAND_SPEED,45
 LAND_SPEED_HIGH,220
+LGR_SERVO_DEPLOY,1100
+LGR_SERVO_RTRACT,1900
+LGR_STARTUP,0
 LOG_BACKEND_TYPE,3
-LOG_DISARMED,1
```

Comment:
I commented on the previous pr. In response, solo logs from Disarm. Therefore, I think that it is better not to delete it.

Reply:
oops.  I keep that turned off on my personal Solo and forgot to change it back for the PR.  Personally, I hate disarmed logging since it makes for a massive log file. But for everyone else, it is often needed for troubleshooting.